app/src/app/components/about-page/about-page.component.ts

```diff
@@ -12,21 +12,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { TestBed, async } from '@angular/core/testing';
-import { NavBarComponent } from './navbar.component';
+import { Component } from '@angular/core';
 
-describe('NavBarComponent', () => {
-  beforeEach(async(() => {
-    TestBed.configureTestingModule({
-      declarations: [
-        NavBarComponent
-      ],
-    }).compileComponents();
-  }));
+enum Diagram {
+  'merge',
+  'overlay',
+  'filter'
+}
 
-  it('creates the navbar component', () => {
-    const fixture = TestBed.createComponent(NavBarComponent);
-    const app = fixture.componentInstance;
-    expect(app).toBeTruthy();
-  });
-});
+@Component({
+  selector: 'app-about-page',
+  templateUrl: './about-page.component.html',
+  styleUrls: ['./about-page.component.scss'],
+})
+export class AboutPageComponent {
+  debugEnabled = false;
+
+  debugMode() {
+    if (this.debugEnabled) {
+      document.body.classList.remove('debug-mode');
+      this.debugEnabled = false;
+    } else {
+      document.body.classList.add('debug-mode');
+      this.debugEnabled = true;
+    }
```

Comment:
Nit: move 'debug-mode' to a constant.

Nit: assign this.debugEnabled = !this.debugEnabled outside of the branching logic.

Reply:
Changed to use a single line